backend/controllers/event_controllers/event_registration_controller.py

```
@@ -0,0 +1,2102 @@
+            reg_open, err = _coerce_to_aware_utc(instance.registration_opens)
+            if reg_open > now:
+                return {"success": False, "msg": "Error! You cannot register for an event before the opening date!"}
+
+        # If registration deadline passed, forbid registration
+        if instance.registration_deadline:
+            reg_dead, err = _coerce_to_aware_utc(instance.registration_deadline)
+            if now >= reg_dead:
+                return {"success": False, "msg": "Error! You cannot register for an event after the registration deadline!"}
+
+    # If the event date has already passed, forbid any update
+    ev_date, err = _coerce_to_aware_utc(instance.date)
```

Ruff (0.14.5)

Unpacked variable `err` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)